prod-sample/sample-modules/frank-lloyd-root/0.0.0/src/config.js

```diff
@@ -18,6 +18,7 @@ import csp from './csp';
 import createFrankLikeFetch from './createFrankLikeFetch';
 
 export default {
+  eventLoopDelayThreshold: Infinity,
```

Comment:
Should this be set to a smaller amount?

Reply:
This is just to prevent the circuit from opening during the integration tests